addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -283,6 +285,53 @@ var Wysiwyg = Widget.extend({
             mediaDialog.on('cancel', this, resolve);
         });
     },
+    changeSelectedTextColor(color, closeColorPicker = false) {
+        if(color === "") {
+            this.editor.execCommand('uncolorText');
+            $(window).trigger('click');
```

Comment:
Why do you trigger a click?

Reply:
I believe it is to close the colorpicker. It is a strange signal to send to close the modal. You cannot do something like `ColorPicker.close()`?

Reply:
To close the colorPicker widget via the documetn click listener